Content.Client/GameObjects/Components/Interactable/ToolComponent.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using Content.Client.UserInterface.Stylesheets;
+using Content.Client.Utility;
+using Content.Shared.GameObjects.Components.Interactable;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+using Robust.Shared.Serialization;
+using Robust.Shared.Timing;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Client.GameObjects.Components.Interactable
+{
+    [RegisterComponent]
+    public class ToolComponent : SharedToolComponent, IItemStatus
+    {
+        private Tool _behavior;
+        private bool _statusShowBehavior;
+
+        [ViewVariables(VVAccess.ReadWrite)] private bool _uiUpdateNeeded;
+        [ViewVariables] public bool StatusShowBehavior => _statusShowBehavior;
```

Comment:
This `StatusShowBehavior` thing should absolutely be a part of multitool, not the base `ToolComponent`.

Reply:
I just removed client-side ToolComponent in favor of client-side MultiToolComponent